channelserver/src/session.rs

```diff
@@ -8,13 +8,13 @@ use actix_web::ws;
 use cadence::{StatsdClient};
 use ipnet::IpNet;
 use maxminddb;
-use uuid::Uuid;
 
+use channelid::ChannelID;
 use logging;
 use meta::SenderData;
 use server;
 
-pub type ChannelName = Uuid;
+pub type ChannelType = ChannelID;
```

Comment:
What value does this typedef provide now that we've got a dedicated `ChannelID` type?

Reply:
Heh, in case we change it again? ;) will remove.